lib/widgets/caregiver/child_list_element.dart

```diff
@@ -0,0 +1,95 @@
+import 'dart:developer';
+
+import 'package:flutter/material.dart';
+
+import 'package:fokus/utils/app_locales.dart';
+import 'package:fokus/utils/theme_config.dart';
+
+import 'package:fokus/widgets/child_wallet.dart';
+
+class ChildListElement extends StatelessWidget {
```

Comment:
This is the same as the reward card so maybe worth generalizing the widget and parameters names?